modules/light-clients/11-beefy/types/update.go

```diff
@@ -208,7 +208,7 @@ func (cs *ClientState) parachainHeadersToMMRProof(beefyHeader *Header) (*mmr.Pro
 		}
 
 		// not a fan of this but its golang
-		var parachainHeads = [32]byte{}
+		var parachainHeads SizedByte32
```

Comment:
And scale encoding/decoding works on these types?

Reply:
Yes ser! Should we include tests for that?

Reply:
would prefer to see tests